app/controllers/idv/jurisdiction_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  class JurisdictionController < ApplicationController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_attempts_allowed
+    before_action :confirm_idv_needed
+    before_action :jurisdiction_form, only: %i[new create]
```

Comment:
What does this need to memoized this in a before action? Can't we just call `jurisdiction_form` where we need it?

Reply:
we can and I would prefer to, but then Rubocop thinks my methods are too long

Reply:
You should be able to call the `jurisdiction_form` method in your controller actions and it will be initialized then, instead of using the instance variable.

Reply:
So...

```ruby
result = jurisdiction_form.submit(jurisdiction_params)
```

instead of 

```ruby
result = @jurisdiction_form.submit(jurisdiction_params)
```

Reply:
The instance variable is used in the view.

Reply:
Gotcha. In that case, let's call the method `set_jurisdiction_form`